src/components/Form.js

```diff
@@ -135,8 +135,29 @@ function Form(props) {
 
             // Validate the input for html tags. It should supercede any other error
             _.each(trimmedStringValues, (inputValue, inputID) => {
+                const foundHtmlTagIndex = inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+                const matchedHtmlTags = inputValue.match(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+                let isMatch = _.some(CONST.WHITELISTED_TAGS, (r) => r.test(inputValue));
```

Comment:
Coming from [this discussion](https://github.com/Expensify/App/pull/25530#discussion_r1308736503). I think we can easily move these two lines right before the 149 line 
```js
if (matchedHtmlTags) {
```